### PR TITLE
Edits Gov V1  Page

### PR DESCRIPTION
**Pull request by filippoweb3**
Rewritten and reorg for better clarity and flow through the sections

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK I took the initiative to review the old gov page, reorg a bit, and add some context where needed to improve the reading flow. Let me know what you think. I will switch to OpenGov next week.

File: docs/learn/learn-governance.md [hunk 1]
```diff
@@ -24,63 +24,66 @@ Learn about the upcoming changes to the governance on
 
 :::
 
-:::caution Upcoming governance changes
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} brings together various novel
+mechanisms, including an amorphous (abstract) state-transition function stored on-chain defined in a
```
Comment (profradha) [inline docs/learn/learn-governance.md]:
```suggestion
mechanisms, including an amorphous (abstract) form of state-transition function stored on-chain defined in a
```

File: docs/learn/learn-governance.md [hunk 2]
```diff
@@ -24,63 +24,66 @@ Learn about the upcoming changes to the governance on
...
-
-:::
-
-To do this, the network brings together various novel mechanisms, including an amorphous
-state-transition function stored on-chain and defined in a platform-neutral intermediate language
-(i.e. [WebAssembly](learn-wasm.md)) and several on-chain voting mechanisms such as referenda with
-adaptive super-majority thresholds and batch approval voting.
+To make any changes to the network, the idea is to compose active token holders and the council
+together to administrate a network upgrade decision. No matter whether the proposal is proposed by
+the public (token holders) or the [Council](#council), it finally will have to go through a
```
Comment (profradha) [inline docs/learn/learn-governance.md]:
```suggestion
the public (token holders) or the [Council](#council), it finally will have to go through a vote on a
```

File: docs/learn/learn-governance.md [hunk 3]
```diff
@@ -91,48 +94,63 @@ Note that this may be different from the absolute number of endorsements; for in
...
+### Council Referenda
+
+Unanimous Council - When all council members agree on a proposal, it can be moved to a referendum
+with a [**negative turnout bias**](#adaptive-quorum-biasing). Briefly, it will require a heavy
+supermajority of _nay_ votes to reject at low turnouts, but as turnout increases towards 100%, it
+will require a simple majority of _nay_ votes to fail (i.e. 51% wins).
 
 Majority Council - When agreement from only a simple majority of council members occurs, the
-referendum can also be voted upon, but it will be majority-carries (51% wins).
+referendum will have a [**neutral turnout bias**](#adaptive-quorum-biasing) with simple
```
Comment (profradha) [inline docs/learn/learn-governance.md]: What is a neutral turnout bias? Bias itself means that it is leaning towards one end or the other. I think it should just be stated as "Simple majority" wins.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**